src/pages/Inventory.page.js

```diff
@@ -1,15 +1,64 @@
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable no-await-in-loop */
 const { BaseSwagLabPage } = require('./BaseSwagLab.page');
 
 export class InventoryPage extends BaseSwagLabPage {
     url = '/inventory.html';
 
-    get headerTitle() { return this.page.locator('.title'); } //
+    get addButton() { return this.page.locator('//button[contains(text(), "Add to cart")]'); }
+
+    get headerTitle() { return this.page.locator('.title'); } // .header_secondary_container - parent
+
+    get productsSortSelect() { return this.page.locator('.product_sort_container'); }
 
     get inventoryItems() { return this.page.locator('.inventory_item'); }
 
-    get addItemToCartBtns() { return this.page.locator('[id^="add-to-cart"]'); }
+    get inventoryItemName() { return this.page.locator('.inventory_item_name'); }
+
+    get inventoryItemPrice() { return this.page.locator('.inventory_item_price'); }
+
+    get addItemToCartButtons() { return this.page.locator('[id^="add-to-cart"]'); }
 
     async addItemToCartById(id) {
-        await this.addItemToCartBtns.nth(id).click();
+        await this.addItemToCartButtons.nth(id).click();
+    }
+
+    async performSorting(value) {
+        await this.productsSortSelect.selectOption(value); // az, za, lohi, hilo
```

Comment:
This comment can be added at line 25. Check how you can do that by using JSDoc - https://www.youtube.com/watch?v=Nqv6UkTROak
Keep in mind for your purposes you don't need to npm install the JSDoc